MultimodalQnA/ui/gradio/multimodalqna_ui_gradio.py

```diff
@@ -316,16 +316,20 @@ def ingest_with_text(filepath, text, request: gr.Request):
         yield (
             gr.Textbox(
                 visible=True,
-                value="Something went wrong!\nPlease click the X button on the top right of the image upload box to reupload your image!",
+                value=f"Something went wrong (server error: {response.status_code})!\nPlease click the X button on the top right of the image upload box to reupload your image!",
             )
         )
         time.sleep(2)
     return
 
 
-def clear_uploaded_file(request: gr.Request):
+def hide_text(request: gr.Request):
     return gr.Textbox(visible=False)
 
+
+def clear_text(request: gr.Request):
```

Comment:
This doesn't need to do anything?

Reply:
The event listener that calls this is using the returned None value by assigning it to the designated output component, which is a Textbox. I agree it looks weird and I could return  `gr.Textbox(value=None)` but it would do the exact same thing.